### ISSUE TITLE
disallow calling super() outside the class constructor

### ISSUE THREAD

**Issue by fega** [eslint/eslint#8898]
## What the rule should do:

disallow calling super() outside the class constructor
it  Warns about a potential error.

## Examples:
```javascript
// BAD
class A extends B {
  method(){
  super()
 }
}
// good
class A extends B {
  method(){
  super.method()
 }
}
```

## Why should this rule be included in ESLint (instead of a plugin):
Is in the core of javascript

**Comment by platinumazure** [comment]
I'm surprised we don't already have something like this. I'm :+1: to this if we don't already have a rule.

**Comment by not-an-aardvark** [comment]
I think this is actually a syntax error, so it's something that should be fixed by acorn (see https://github.com/ternjs/acorn/issues/448)

**Comment by mysticatea** [comment]
Yes, it's a syntax error, so it should be caught by parsers.

**Comment by not-an-aardvark** [comment]
Closing because this should be addressed at the parser level. I don't think it would make sense to add a rule to core for it in the meantime, because the rule would become useless when the parser bug is fixed.

**Comment by fega** [comment]
yep, I agree, thanks :)
